proxy/envoy/config.go

```diff
@@ -167,8 +167,15 @@ func buildHTTPListener(routeConfig *HTTPRouteConfig, ip string, port int, rds bo
 		config.RouteConfig = routeConfig
 	}
 
+	var sslContext *SSLContext
+	// Build the sslContext for http inbound listener (only inbound listener does not use rds)
+	if !rds && mesh.EnableAuth {
```

Comment:
we need another parameters to use auth then. RDS can be used for inbound traffic (weird, but possible).